src/App.tsx

```diff
@@ -15,36 +17,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  SORT_ALPHABET = 'SORT_ALPHABET ',
+  SORT_LENGTH = 'SORT_LENGTH',
+}
+
+type Props = {
+  sortField: string;
```

Comment:
The `sortField` property in the `Props` type is currently a `string`. It would be better to type it as `SortType` to ensure only valid sort types are used, preventing potential runtime errors.